packages/appstore/src/components/wallets-banner/wallets-banner.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import WalletsBannerUpgrade from './wallets-banner-upgrade';
+
+type TWalletsMigrationStatus = 'ineligible' | 'eligible' | 'in_progress' | 'migrated' | 'failed';
```

Comment:
Maybe it will be used for different places and situations, so It's better to move it to the types folder.

Reply:
@hamid-deriv right now we don't have BE so it's not final type, I'll change it in a future. But agree with you it must be in types folder, I'll move it